vocabularies/geoadmin-feature-types.ttl

```diff
@@ -0,0 +1,69 @@
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Sub-Block"@en .
+
+<http://linked.data.gov.au/def/geofeatures> skos:definition "This vocabulary lists types of geological features relevant to the duties of the Geological Survey of Queensland. It is automatically derived from GSQ's Geologic Features Ontology (http://linked.data.gov.au/def/geofeatures)"@en ;
+    skos:prefLabel "Geologic Feature Types"@en .
+
+<https://orcid.org/0000-0002-8742-7730> a owl:NamedIndividual,
+        sdo:Person ;
+    sdo:affiliation [ a sdo:Organization ;
+            sdo:name "SURROUND Australia Pty Ltd" ;
+            sdo:url <https://surroundaustralia.com> ] ;
+    sdo:email <mailto:nicholas.car@surroundaustralia.com> ;
+    sdo:name "Nicholas J. Car" .
+
+<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
+    sdo:email <mailto:geological_info@dnrme.qld.gov.au> ;
+    sdo:name "Geological Survey of Queensland" ;
+    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
+
+<http://linked.data.gov.au/def/geoadminfeatures> a owl:Ontology,
+        skos:ConceptScheme ;
+    dcterms:created "2019-12-04"^^xsd:date ;
+    dcterms:creator <http://linked.data.gov.au/org/gsq>,
+        <https://orcid.org/0000-0002-8742-7730> ;
+    dcterms:description """This ontology describes a class hierarchy relevant to the administrative duties of the Geological Survey of Queensland.
+
+Within a class hierarchy starting at GeoSPARQL Ontology's `Feature` class, this ontology defines an `Administrative Feature` subclass and further subclasses of it, some with class interrelations. The classes defined include `Block`, `Permit` etc., some of which are defined in Queensland or Australian legislation.
+
+This ontology is part of the Geological Survey of Queensland's set of *component* ontologies that together provide definitions for all of GSQ's data. Of particular importance for this ontology is GSQ's Geological Features Ontology which defines multiple classes of geological feature."""@en ;
+    dcterms:modified "2020-04-15"^^xsd:date ;
+    dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+    dcterms:title "GSQ Geological Administrative Features Ontology"@en ;
+    owl:imports <http://www.opengis.net/ont/geosparql> ;
+    skos:hasTopConcept <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    sdo:codeRepository <https://github.com/geological-survey-of-queensland/gsq-geoadmin-features-ont> .
+
+<http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> a skos:Concept ;
```

Comment:
I don't understand what this concept is doing here.

Reply:
The order's not at all important in this graph: it's just been serialised after the ConceptSscheme due to internal graph ordering.